src/content/guides/caching.md

```diff
@@ -108,7 +109,9 @@ W> Output may differ depending on your current webpack version. Newer versions m
 
 ## Extracting Boilerplate
 
-As we learned in [code splitting](/guides/code-splitting), the [`SplitChunksPlugin`](/plugins/split-chunks-plugin/) can be used to split modules out into separate bundles. webpack provides an optimization feature which does split out runtime code into a separate chunk(s) according to the options provided, simply use [`optimization.runtimeChunk`](/configuration/optimization/#optimization-runtimechunk) set to `single` for creating one runtime bundle:
+Boilerlate manifest code are typically we you will find at the top of you bundle file, they are auto generated and will change each time your code does, which result to massive added wieght to your overall bundle file which can be very costy in term of loading time.
```

Comment:
Typo `weight`.

Reply:
I don't understand first sentence at all: `Boilerlate manifest code are typically we you will find at the top of you bundle file, ...`. What are you trying to say?